test/functional/rpc_mempool_entry_fee_fields_deprecation.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test deprecation of fee fields from top level mempool entry object"""
```

Comment:
Any reason you didn't use the existing test file, rpc_deprecated.py? I don't personally have an opinion on whether it would be better, but it looks like a good place to put these tests too.

Reply:
i saw `rpc_deprecated.py` and wasn't sure why it was there? right now it doesn't test anything. i then saw `rpc_addresses_deprecation.py` was its own file, so i followed that one as an example.

i guess the advantage of putting them there is it might help things run faster than running individual test files for each deprecated rpc. i can put it there and then move `rpc_addresses_deprecation.py` to `rpc_deprecated.py` as a follow-up pr

Reply:
I assumed that rpc_deprecated.py there to test RPC deprecation stuff and address/reqSigs tests are in a separate file because they need the wallet to be built. So they can't be moved to rpc_deprecated.py either.

Reply:
actually, ive talked myself out of this. if we have individual files, they can run in parallel and if we put all the deprecated tests under the same class it could be really annoying later on as you are adding a new test. you'd have to wait for N deprecation tests to run each time as you're iterating on your test, which starts to impact productivity if it takes 30 ~ 45 seconds to run through all the tests each time.

what are your thoughts, @glozow ?

Reply:
> I assumed that rpc_deprecated.py is there to test RPC deprecation stuff and address/reqSigs tests are in a separate file because they need the wallet to be built

makes sense. i read through the comments on `rpc_deprecated.py` and it seems focused on testing that a whole rpc command is deprecated. i think it works well for that and we can consolidate rpc command deprecation's there.

this is test slightly different in that an rpc is not being deprecated (meaning we cant just check for an rpc not found error), but fields on an rpc response are being deprecated, which makes the setup for the test a bit more complicated. for this reason, id suggest leaving this in a separate file

Reply:
I don't think it matters that much :shrug: either way makes sense to me. I'd imagine that the extra overhead of spinning up test frameworks for separate test files overrides the gains from running them in parallel.